src/main/resources/application.properties

```diff
@@ -3,6 +3,7 @@ server.port=8080
 workspace.maxStairwayThreads=4
 workspace.resourceId=mc-terra-workspace-manager
 workspace.stairwayTimeoutSeconds=1800
+workspace.stairwayPollingIntervalSeconds=1
```

Comment:
was this previously 10? I'm wondering if polling every second is too fast.

Reply:
It was previously 10s - it turns out 10s is actually too slow, and this is much closer to how long things actually take. It turns out that ~2/3 of our test runtime was waiting for this polling, and this change alone cuts tests from ~7 minutes to ~3 minutes.
That said, it may be too short for long-running operations, like creating stuff on clouds. It's probably worth revisiting then, but the performance boost now seems worth it.

Reply:
Let's keep an eye on this. I think the best approach in the long run might be to introduce backoff into the polling strategy (maybe not exponential, maybe a linear backoff?) so we catch completion of fast flights quickly, but don't overwhelm slow flights with lots of poll requests.

And then ... polling for Stairway flight status with backoff is almost certainly reusable across services, so it should go into a shared library somewhere. Nothing necessary for this PR; let's follow up with another story.

Reply:
Sounds good, created [AS-467] to track that. It's probably also worth adding that functionality to the template repo, which also has a JobService.